_posts/2016-05-01-paxos.html

```diff
@@ -79,13 +79,15 @@
 
   <h5>Distributed FSM</h5>
 
-  <p>Distributed FSM has several endpoints with the execute service, but it doesn't matter which endpoint you choose because the system behaves exactly as a system with several endpoints but leading to the same (non-distributed) FSM. The only difference that the distributed version runs on <span class="num">2n+1</span> nodes and tolerates up to <span class="num">n</span> failures.</p>
+  <p>Distributed FSM has several endpoints <!-- define endpoints --> with the <code>execute</code> service, but it doesn't matter which endpoint you choose because the system behaves exactly like a system with several endpoints but leading to the same (non-distributed) FSM. <!-- refactor this whole sentence, perhaps split it to 2 or more smaller sentences --> The only difference is that the distributed version runs on <span class="num">2n+1</span> nodes and tolerates up to <span class="num">n</span> failures.</p>
 
```

Comment:
In general, this paragraph is not very understandable, especially the first sentence needs refactoring.